core/orchestrator.go

```diff
@@ -192,8 +192,11 @@ func (orch *orchestrator) PriceInfo(sender ethcommon.Address) (*big.Rat, error)
 		return nil, err
 	}
 	// pricePerPixel = basePrice * (1 + 1/ txCostMultiplier)
-	overhead := new(big.Rat).Add(big.NewRat(1, 1), new(big.Rat).Inv(txCostMultiplier))
-	return new(big.Rat).Mul(orch.node.PriceInfo, overhead), nil
+	price := new(big.Rat).Mul(orch.node.PriceInfo, new(big.Rat).Add(big.NewRat(1, 1), new(big.Rat).Inv(txCostMultiplier)))
```

Comment:
Can we keep the `overhead` variable to make it a bit clearer why we're doing the calculation here?

Reply:
Sure !